Frameworks/Foundation/NSDictionary.mm

```diff
@@ -99,8 +99,11 @@ - (NSArray*)allObjects {
         return nil;
     }
 
-    StrongId<NSArray> keyArray = [_keyEnumerator allObjects];
-    StrongId<NSMutableArray> toRet = [NSMutableArray new];
+    StrongId<NSArray> keyArray;
+    keyArray.attach([_keyEnumerator allObjects]);
```

Comment:
>keyArray.attach([_keyEnumerator allObjects]) [](start = 4, length = 44)

This will almost certainly explode later; allObjects returns an autoreleased and attach does not add a +1.